apps/visibleplanets/visibleplanets.star

```diff
@@ -363,9 +394,9 @@ def get_schema():
                 id = "planet",
                 name = "Planet",
                 desc = "Pick the planet you want to track.",
-                icon = "satelliteDish",  #skyatlas #moon, globe, mars, marsandvenus satelliteDish
+                icon = "globe",  #skyatlas #moon, globe, mars, marsandvenus satelliteDish
                 options = planet_options,
-                default = planet_options[0].value,
+                default = planet_options[5].value,
```

Comment:
Just an FYI, anyone with this app installed with default values will see this change once this app is updated in our backend